pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -91,6 +91,18 @@ self: super: {
     };
   };
 
+  # do not auto-update this one, as the name clashes with calendar-vim
```

Comment:
In that case we should have some way for the user to distinguish the two. Maybe it was a mistake to have the plugins in nixpkgs without namespaces.

Ideally you'd implement alias support in our plugin update infrastructure (`calendar-vim as mattn-calendar-vim` in `plugin-list`), but if you don't want to do that we should at least choose a more descriptive name.

Reply:
I'm fine with adding that as an alias in `plugin-list`; should the package definition stay where it is in `overrides.nix`? If it's added to `vim-plugin-list` I'm assuming it'll have a naming conflict with `calendar-vim`.

Reply:
No I meant that if you want to go down that road, you'd have to adapt `update.py` to work with statements like `calendar-vim as mattn-calendar-vim` in `plugin-list`. That way no override would be necessary and auto updates would work.

Reply:
Ah, gotcha. I can look into that, seems like it would be a useful feature to have. Would it be best to put this PR on hold and merge that in separately?

Reply:
Whatever way you prefer. As long as the change to `update.py` is in a separate commit that is fine by me. If you want to do a manual override as a stopgap first that is also okay.

Reply:
Great, sounds good to me.

Reply:
Okay, ping me if you do decide to open a different PR for it.

Reply:
@timokau Just pushed changes to `update.py` to add alias support; see the first commit in the PR. Also reupdated plugins in the update commit, and moved `vim-docbk-snippets` from `overrides.nix` into `vim-plugin-names` since it works with the alias support :)